src/sentry/models/event.py

```diff
@@ -150,13 +150,12 @@ def get_hashes(self):
         information if available.  Grouping hashes will take into account
         fingerprinting and checksums.
         """
-        from sentry.event_hashing import calculate_event_hashes
         # If we have hashes stored in the data we use them, otherwise we
         # fall back to generating new ones from the data
         hashes = self.data.get('hashes')
         if hashes is not None:
             return hashes
-        return calculate_event_hashes(self)
+        return filter(None, [x.get_hash() for x in self.get_grouping_variants().values()])
```

Comment:
This should probably return `list(filter(...))` or be called `iter_hashes` for consistency, just to be sure.